src/Report/main.tex

```diff
@@ -394,12 +394,21 @@
                 Um einen Einblick in die Sicht der Nutzer auf Unternehmen zu bekommen wurden die Endanwender befragt, welches die drei signifikantesten Gründe für den Einsatz von \gls{oss} in Unternehmen sind. Dabei wurden die gleichen Kategorien wie in der Eingangs erwähnten Studie in einer Multiple-Choice Frage abgefragt.
 
             \paragraph{Privat}
-                Zusätzlich ist von Interesse, was die privaten Gründe für die Nutzung von Open Source Anwendungen sind. Dazu gab es in der Umfrage ein Freifeld, wo nach den drei wichtigsten Gründen für den privaten Einsatz von \gls{oss} gefragt wurde. Diese wurden anschließend grob kategorisiert\footnote{Siehe Tabelle \ref{table:categories} auf Seite \pageref{table:categories}.} und anschließend den Kategorien der Schweizer Studie zugeordnet.
+                Zusätzlich ist von Interesse, was die privaten Gründe für die Nutzung von Open Source Anwendungen sind. Dazu gab es in der Umfrage ein Freifeld, wo nach den drei wichtigsten Gründen für den privaten Einsatz von \gls{oss} gefragt wurde. Diese wurden grob kategorisiert\footnote{Siehe Tabelle \ref{table:categories} auf Seite \pageref{table:categories}.} um eine Auswertung möglich zu machen. Das Ergebnis ist in Abbildung \ref{figure:usageRankingPrivate} dargestellt. Dabei sind die Anteile nicht die Anteile der Nutzer, die diese Kategorie als Nutzungsgrund angegeben haben, sondern die Anteile an der Gesamtheit aller angegebenen Begriffe. Da jeder Nutzer genau 3 Begriffe angeben musste, ist die Summe aller Anteile $300\%$. Da nur sehr selten von einem Nutzer mehr als ein Begriff aus einer Kategorie angegeben wurde, lassen sich die Anteile dennoch annäherungsweise als die Anteile als Nutzer interpretieren.
+                
+                Auffällig ist, dass der Aspekt Kosten mit einem großen Abstand (ca. drei mal so oft) sehr häufig angegeben wurde.
```

Comment:
`drei mal so oft` klingt nicht so schön. Evtl. `dreifach` o.ä.

Reply:
Ich weiß nicht wirklich, wie man `dreifach` in den Satz einbauen soll.